src/ripple/app/tx/impl/PayChan.cpp

```diff
@@ -348,6 +354,9 @@ PayChanClaim::preflight (PreflightContext const& ctx)
     if (! ctx.rules.enabled(featurePayChan))
         return temDISABLED;
 
+    if (ctx.rules.enabled(fix1543) && ctx.tx.getFlags() & tfUniversalMask)
+        return temINVALID_FLAG;
```

Comment:
This should be removed - claim can take flags. See line 383.

Reply:
Then we need to define those flags. I'll upgrade.